src/OrganisationRegistry.Api/Backoffice/Admin/Configuration/ConfigurationController.cs

```
@@ -98,9 +94,9 @@ public async Task<IActionResult> Put([FromServices] ConfigurationContext context
             var configurationValue = context.Configuration.Single(x => x.Key == id);
             configurationValue.Value = internalMessage.Body.Value;
             configurationValue.Description = internalMessage.Body.Description;
-            context.SaveChanges();
+            await context.SaveChangesAsync();
 
-            return OkWithLocation(Url.Action(nameof(Get), new { id = internalMessage.Key }));
+            return OkWithLocation(Url.Action(nameof(Get), new { id = internalMessage.Key }) ?? "");
```

I'd rather we make this throw an exception than make it fail silently